AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -126,6 +126,7 @@ tylko.com##.ty-settings
 backmarket.de#$#div[data-test="cookies-modal-v2"] { display: none!important; }
 backmarket.de#$#body { overflow: visible!important; }
 puzzlegarage.com##div[class^="modal--cookies"]
+britishairways.com#$#body { overflow: auto !important; }
```

Comment:
If the `body` or `html` overflow value of needs to be changed, all of the rules have to be written via `#$#` syntax - fixed directly.